features/admin.applications.v1/components/settings/info.tsx

```diff
@@ -105,7 +105,13 @@ export const Info: FunctionComponent<InfoPropsInterface> = (
     /**
      * Index of the protocols tab.
      */
-    const PROTOCOLS_TAB_INDEX: number = 1;
+    let PROTOCOLS_TAB_INDEX: number;
+
+    if (templateId === ApplicationManagementConstants.CUSTOM_APPLICATION_OIDC) {
+        PROTOCOLS_TAB_INDEX = 1;
+    } else {
+        PROTOCOLS_TAB_INDEX = 2;
+    }
```

Comment:
Shall we use tabId based approach for navigating to application tabs?

Reply:
With the above search param based navigation, we can remove this logic, can't we?

Reply:
Fixed with 66db63647fd006555a02f3781c73a0c098394830